detail/detail.js

```diff
@@ -3,7 +3,24 @@ import { renderDogDetail } from '../render-utils.js';
 
 const dogDetailContainer = document.getElementById('dog-detail-container');
 
+
+
+
+
 // on load
```

Comment:
On window load of the detail page, it grabs the param by its id and then renders a dog based on their id in that table the will append that dog to the container which then displays the dog's details by it the corresponding id.